datajunction-server/datajunction_server/api/djql.py

```diff
@@ -0,0 +1,91 @@
+"""
+Data related APIs.
+"""
+
+from typing import Optional
+
+from fastapi import APIRouter, Depends, Request
+from sqlmodel import Session
+from sse_starlette.sse import EventSourceResponse
+
+from datajunction_server.api.helpers import build_sql_for_dj_query, query_event_stream
+from datajunction_server.models.query import QueryCreate, QueryWithResults
+from datajunction_server.service_clients import QueryServiceClient
+from datajunction_server.utils import get_query_service_client, get_session
+
+router = APIRouter()
+
+
+@router.get("/djql/data", response_model=QueryWithResults)
```

Comment:
I like how you referred to it in the docstring as "DJ SQL" since DJQL implies it's some query language other than SQL. So maybe the endpoints should also be `/djsql/...`